### PR TITLE
aging.csl

### PR DESCRIPTION
**Pull request by joseluis0988**

### CODE REVIEW

**Comment by libora** [comment]
@joseluis0988 add this line into info section:
`<rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>`

**Comment by rmzelle** [comment]
@joseluis0988, also please add a "template" link if you can provide one. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

File: aging.csl [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    <terms>
+      <term name="collection-editor" form="long">
+        <single>Series editor</single>
+        <multiple>Series editors</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name delimiter-precedes-last="always" et-al-use-first="13" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```
Comment (rmzelle) [inline aging.csl]: `et-al-use-first="13"`

Is this intentional? Or are the settings `et-al-min="31" et-al-use-first="30"` on `<bibliography>` the correct ones?

**Comment by rmzelle** [comment]
Closing because of inactivity. @joseluis0988, if you get back to us we can just reopen this pull request.

**Pull request closed without merging**